.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,20 @@
+settings:
+  jira_project_key: "OPENG"
+  status_mapping:
+    opened: Untriaged
+    closed: done
+    not_planned: rejected
+    
+  components:
+    - karma
+    - team-core
+
+  labels:
+    - 25.04
+      
+  add_gh_comment: false
+  sync_description: false
+  sync_comments: false
+
+  label_mapping:
+    "Type: Enhancement": Story
```

Comment:
Any reasons why it's different from https://github.com/canonical/istio-k8s-operator/pull/33/files#diff-eee108ea49e324e51ab610c63c844d18b5b59c80bf21fd9a5b86dbecd8ad1983R20 ?